gdpr/docs/tunnistamo-profile-configuration.md

```diff
@@ -0,0 +1,53 @@
+# Helsinki-Profile with Tunnistamo
+
+## OIDC provider clients
+
+Model: `oidc_provider.models.Client`
+
+### Helsinki-Profile API
+
+Important fields with their values:
+
+```python
+{
+    "name": "Helsinkiprofile",
+    "client_type": "confidential",
+    "client_id": "https://api.hel.fi/auth/helsinkiprofile",
+    "client_secret": "52625d8d2661df1710076956808725588deabb2d9902b06003d78764",
```

Comment:
Is this a testing environment or local environment secret?

Reply:
it's my local (it is automatically generated when the client type is set to confidential).